spec/javascripts/unit/AnalyticsSpec.js

```diff
@@ -177,6 +177,21 @@ describe("GOVUK.Analytics", function () {
         'transport': 'beacon'
       }]);
     });
+
+    it('sends the right event when an opportunity data download link is clicked', function() {
```

Comment:
Hmm, I guess this test should cover the thing I was worried about earlier.  But it only checks `window.ga.calls.first().args` - should we also check that no other calls are made to GA - e.g. ones that say the supplier list has been downloaded?
This seems like a possible drawback of many of these existing javascript tests - they thest that one thing has happened that we want, but not that a bunch of unwanted things haven't happened along with the one good thing.

Reply:
But I have no real idea about Javascript, so maybe this is all fine.

Reply:
This is a good point, I will add a test for not receiving more calls than needed and see what happens :).

Reply:
My Javascript is also super rusty 😬 . @katstevens was my hero yesterday and and helped me get back to it after 8-9 months break 😆